lua/gruvbox-baby/colors.lua

```diff
@@ -92,6 +92,15 @@ function M.config(config)
     }
     colors = vim.tbl_extend("force", colors, transparent)
   end
+
+  if config.overrides then
+    for override_color, new_color in pairs(config.overrides) do
+      if new_color:match("^%a") then -- use existing palette color
+        new_color = colors[new_color]
+      end
```

Comment:
I think is better if we remove this option, we already provide a way for getting the colors with `require("gruvbox-baby.colors").config()`, so if anyone wants to use some of the existing colors they can do it that way. Also we don't have to deal with strings that don't match the keys on the colors dictionary.

Reply:
yeah that makes sense.

> Also we don't have to deal with strings that don't match the keys on the colors dictionary.

That's probably why you didn't use this method on the highlights config option?

Reply:
correct